core-bom/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'io.micronaut.build.internal.bom'
+}
+
+group projectGroupId
+version projectVersion
+
+repositories {
+    mavenCentral()
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
+}
+
+micronautBom {
+    extraExcludedProjects = [
+            "benchmarks",
+            "inject-test-utils",
+            "inject-java-test",
+            "inject-kotlin-test",
+            "inject-groovy-test"
```

Comment:
These were actually in the previous BOM

Reply:
They were indeed, but I thought those were internal so a mistake. Isn't it the case?